homeassistant/components/ridwell/strings.json

```diff
@@ -24,5 +24,17 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "next_pickup": {
+        "name": "Next Ridwell pickup"
```

Comment:
Should Ridwell be removed from this entity name? I feel like that is kinda implied by the integration itself already?

Reply:
Good point